package.json

```diff
@@ -1,27 +1,24 @@
 {
     "private": true,
     "scripts": {
-        "dev": "npm run development",
-        "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-        "watch": "npm run development -- --watch",
-        "watch-poll": "npm run watch -- --watch-poll",
-        "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
-        "prod": "npm run production",
-        "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
+        "dev": "vite",
+        "build": "vite build"
     },
     "devDependencies": {
-        "axios": "^0.21.0",
+        "@vitejs/plugin-vue2": "^2.2.0",
+        "axios": "^0.21.4",
         "bootstrap": "^4.0.0",
-        "cross-env": "^5.1",
-        "jquery": "^3.2",
-        "laravel-mix": "^5.0.1",
+        "cross-env": "^6.0.3",
+        "jquery": "^3.4.1",
+        "laravel-vite-plugin": "^0.7.4",
         "lodash": "^4.17.19",
         "popper.js": "^1.12",
         "resolve-url-loader": "^3.1.0",
-        "sass": "^1.24.5",
+        "sass": "^1.49.7",
         "sass-loader": "^8.0.2",
-        "vue": "^2.6.11",
-        "vue-template-compiler": "^2.6.11"
+        "vite": "^4.2.1",
+        "vue": "^2.7.14",
+        "vue-template-compiler": "^2.7.14"
     },
     "dependencies": {
         "vee-validate": "^3.3.0"
```

Comment:
Hi @AmooAti,

Can you check these dependencies, as I am installing I am seeing a deprecated warning?

I feel that if we are upgrading Laravel then dependencies should also be up to date.

Reply:
Hi, @devansh-webkul 
Yes, I didn't see depreciation warnings cause I already installed them.
I'll upgrade deprecated packages.
I wanted to upgrade Vue to version 3 in this file but wanted to know if it may cause some difficulties with Vue 2, which is used in Bagisto core packages. (user or third parties may want to extend a core functionality)

Reply:
I think root path dependencies are not even used because we already added Vue JS dependencies in the individual package.

Can you just try to clean it a bit further by removing the dependencies?

Reply:
The default root path structure can follow the same laravel like pattern. Inside the package level we will handle it internally.